build/scenarios.sh

```diff
@@ -37,7 +37,7 @@ htmlJobs="-j $ROOT/src/Benchmarks/benchmarks.html.json"
 jsonJobs="-j $ROOT/src/Benchmarks/benchmarks.json.json"
 multiQueryJobs="-j $ROOT/src/Benchmarks/benchmarks.multiquery.json"
 httpClientJobs="-j $ROOT/src/Benchmarks/benchmarks.httpclient.json"
-signalRJobs="-j https://raw.githubusercontent.com/aspnet/SignalR/release/2.2/benchmarkapps/BenchmarkServer/signalr.json -t SignalR -r signalr --projectFile benchmarkapps/BenchmarkServer/BenchmarkServer.csproj"
+signalRJobs="-j https://raw.githubusercontent.com/aspnet/AspNetCore/master/src/SignalR/benchmarkapps/BenchmarkServer/signalr.json -t SignalR -r aspnetcore --projectFile src/SignalR/benchmarkapps/BenchmarkServer/BenchmarkServer.csproj"
```

Comment:
the `-r aspnetcore` parameter is useless, it's just overwritting what is in the json file. same for the `--projectfile`.